modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc

```diff
@@ -0,0 +1,100 @@
+= Using Your Own Prometheus Server to Consume Couchbase Metrics
+:description: With Couchbase 7.x, it's not necessary to write a custom Prometheus Exporter.This page will demonstrate how to enable a Prometheus server to read from Couchbase's Prometheus-ready endpoints.
+
+[abstract]
+{description}
+
+== Prerequisites
+
+In this tutorial, we're going to use a https://www.docker.com/get-started[Docker] instance of https://prometheus.io/[Prometheus] to build our consumer. Make sure that you have Docker installed and running on your machine. You should also have an available Couchbase cluster with at least one running node.
```

Comment:
For ease of reviewing we tend to put new sentences on a newline.
https://docs.couchbase.com/home/contribute/pages.html#ventilate

There might be a few other sections where this needs to be updated, but as an example:

```suggestion
In this tutorial, we're going to use a https://www.docker.com/get-started[Docker] instance of https://prometheus.io/[Prometheus] to build our consumer.
Make sure that you have Docker installed and running on your machine.
You should also have an available Couchbase cluster with at least one running node.
```

Reply:
Ease of reviewing, yes - in particular with diffs when a page is updated later.
(have fixed in my gardening edit).